Lib/openbakery/profiles/name.py

```diff
@@ -439,7 +438,7 @@ def com_google_fonts_check_family_naming_recommendations(ttFont):
                 {
                     "field": "PostScript Name",
                     "value": string,
-                    "rec": ("May contain only a-zA-Z0-9" " characters and an hyphen."),
+                    "rec": ("May contain only a-zA-Z0-9" " characters and a hyphen."),
```

Comment:
This is a concatenation of 2 strings on a single line. I suggest replacing it by a single string:
```
 "rec": ("May contain only a-zA-Z0-9 characters and a hyphen."),
```

Reply:
Ugh. Thanks to `black` for introducing that one 🙄 
There are a few others as well, thanks for catching these.